packages/react-core/src/components/OverflowMenu/examples/OverflowMenuBreakpointOnContainer.tsx

```diff
@@ -85,13 +86,21 @@ export const OverflowMenuBreakpointOnContainer: React.FunctionComponent = () =>
           <OverflowMenuControl>
             <Dropdown
```

Comment:
This example still needs the `onOpenChange` callback passed in with a `setIsOpen` call once the issue involving the incorrect Dropdown being used gets resolved (I believe that was mentioned as a current issue? Using the `Dropdown as DropdownNext` alias like other examples seems like it might be a temporary fix).